.couch_email_authrc-dist

```diff
@@ -0,0 +1,16 @@
+host = 127.0.0.1
+port = 3000
+couchDbBaseUrl = http://admin:secret@localhost:5984
+usersDb = _users
+sessionTimeout = 1
```

Comment:
This is the token expiration time in seconds.  We should either leave it out, or use a sensible default.

Btw., the name `sessionTimeout` is misleading.  We should rename this configuration to something more self-explanatory like e.g. `tokenExpiryTime`.  I'll address that in a separate issue.